NDTensors/src/lib/Sectors/src/category_definitions/su.jl

```diff
@@ -26,6 +30,37 @@ function dimension(s::SU)
   return Int(d)
 end
 
+function dual(s::SU)
+  l = label(s)
+  nl = ((reverse(cumsum(l[begin:(end - 1)] .- l[(begin + 1):end]))..., 0))
+  return typeof(s)(nl)
+end
+
+# display SU(N) irrep as a Young tableau with utf8 box char
+function Base.show(io::IO, ::MIME"text/plain", s::SU)
+  l = label(s)
+  if l[1] == 0  # singlet = no box
+    println("●")
```

Comment:
I believe you should be using `println(io, ...)` so the output gets sent to the right I/O stream.

Reply:
I.e. https://docs.julialang.org/en/v1/base/io-network/#Base.show-Tuple{IO,%20Any,%20Any}.